build.sh

```diff
@@ -1,7 +1,50 @@
 #!/bin/bash
 
-NGINX_VERSION=${NGINX_VERSION:-1.25.1}
+#########################################################################
```

Comment:
I added a validation check on the distros for which we have build resources for.